cognite/extractorutils/uploader/_base.py

```diff
@@ -56,8 +60,16 @@ def __init__(
         self.upload_queue_size = 0
 
         self.trigger_log_level = _resolve_log_level(trigger_log_level)
+
         self.logger = logging.getLogger(__name__)
 
+        log_level = logging.getLevelName(self.logger.getEffectiveLevel())
+
+        httpx_log_level = resolve_log_level_for_httpx(log_level)
+        httpx_logging_configuration = generate_config_for_http_logging(httpx_log_level)
+
+        logging.config.dictConfig(httpx_logging_configuration)  # type: ignore
```

Comment:
I really don't love this approach. It would be much better to actually build the config programatically, no? Here you just dump a dict and type: ignore it, we really should avoid that kind of logic whenever possible.

Reply:
okay fair, can do. 

The reason for type: ignore is that mypy thinks logging.config is not a valid attribute.

Reply:
I found out that we are configuring all of the logger stuff in elements.py so was able to configure it as part of that. I think that should do the job. Do let me know if this looks good.